lib/ts/recipe/thirdpartypasswordless/recipe.ts

```diff
@@ -50,6 +57,10 @@ export default class Recipe extends RecipeModule {
 
     apiImpl: APIInterface;
 
+    emailDelivery: EmailDeliveryIngredient<TypeThirdPartyPasswordlessEmailDeliveryInput>;
+
+    isInServerlessEnv: boolean;
```

Comment:
Why does this need to be an instance var?

Reply:
Need the `isInServerlessEnv` boolean for the BackwardCompatibilityService. BackwardCompatibilityService fetches this from the recipeInstance. I can pass this in the `validateAndNormaliseUserInput` function, but to be consistent across all other recipes, I've kept it here. Most of the other recipes have this property already

Reply:
Sounds good.